helm/kanister-operator/values.yaml

```diff
@@ -89,3 +89,4 @@ tolerations: []
 #
 # node labels for pod assignment. Evaluated as template
 nodeSelector: {}
+
```

Comment:
are we not adding any new fields in values.yaml for `.Values.controller.containerSecurityContext`? Is it already part of the values.yaml?

Reply:
@viveksinghggits no we are not adding . Please check this [discussion] (https://github.com/kanisterio/kanister/pull/3075#discussion_r1776908764)

and regarding for nodeSelector . It was already there before this PR. i added the new line based on this [recommendation] (https://github.com/kanisterio/kanister/pull/3075#pullrequestreview-2330976614)

Reply:
@anishbista60 I think there was a miscommunication. I was not talking about removing entire helm field. I was talking about removing the things that you had commented out https://github.com/kanisterio/kanister/pull/3075#discussion_r1770596824 and removing the default values that you had put. We would still the new helm fields that the PR is introducing.